src/main/java/seedu/address/commons/util/XmlUtil.java

```diff
@@ -0,0 +1,41 @@
+package seedu.address.commons.util;
+
+import java.io.StringReader;
+import java.io.StringWriter;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.transform.OutputKeys;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+
+import org.w3c.dom.Document;
+import org.xml.sax.InputSource;
+
+/**
+ * Converts a XML Document to String and vice versa.
+ * Code modified from https://www.journaldev.com/1237/java-convert-string-to-xml-document-and-xml-document-to-string
+ */
+public class XmlUtil {
+
+    public static String convertDocumentToString(Document doc) throws TransformerException {
+        TransformerFactory tf = TransformerFactory.newInstance();
+        Transformer transformer = tf.newTransformer();
+        transformer.setOutputProperty(OutputKeys.OMIT_XML_DECLARATION, "yes");
+        StringWriter writer = new StringWriter();
+        transformer.transform(new DOMSource(doc), new StreamResult(writer));
+        String output = writer.getBuffer().toString();
+        return output;
+    }
+
+    private static Document convertStringToDocument(String xmlStr) throws Exception {
```

Comment:
`throws Exception` do we have a more specific type here?

Reply:
There's a couple: `javax.xml.parsers.ParserConfigurationException`, `org.xml.sax.SAXException`, `java io.IOException`